GunShipAlpha/src/Game.java

```diff
@@ -32,9 +33,19 @@ private void loadSprite() {
     }
 
     private void startGameLoop() {
-        Timer gameLoop = new Timer(16, e -> {
-            moveEnemies();
-            repaint();
+        Timer gameLoop = new Timer(30, e -> {
```

Comment:
why 16 vs 30? magic numbers bad.

Reply:
because the enemies were moving too fast and I guess it was a lazy way of slowing them down, 16 is the delay to get 60 frames per second